Mage/src/main/java/mage/abilities/effects/common/continuous/GainAbilityAttachedEffect.java

```diff
@@ -15,6 +15,7 @@
     protected Ability ability;
     protected AttachmentType attachmentType;
     protected boolean independentEffect;
+    protected String targetObjectName;
```

Comment:
Missing `targetObjectName` in copy constructor
* `public GainAbilityAttachedEffect(final GainAbilityAttachedEffect effect)`

Reply:
Added in commit a6a2511. Thanks!